packages/core/src/services/angle-utils.ts

```diff
@@ -82,6 +82,13 @@ export function polarToCartesianCoords(
 ): Point {
 	const x = r * Math.cos(a) + t.x;
 	const y = r * Math.sin(a) + t.y;
+
+	// NaN is rendered at coordinate 0 in browsers
+	// By setting it to 0, further operations can be performed
+	if (isNaN(x) || isNaN(y)) {
```

Comment:
shouldn't this just set x to 0 if it's NaN, and same for y?

are they always both NaN?

Reply:
I don't think it really matters, if either x or y equals NaN it will render at position 0. But I'll add separate conditions for both x and y.